### PR TITLE
[RFR]: Export component Loading

### PR DESCRIPTION
**Pull request by tkvw**
I think this component should be exported.

### CODE REVIEW

**Comment by fzaninotto** [comment]
Can you elaborate? In which case should end users use it?

I worry about it because every public component should be documented, maintained, and kept BC. We try to keep the public API surface small.

**Comment by tkvw** [comment]
For consistency in the loading component, it allows the user to use it if needed. I use it for startup of the app (reauthentication and locale loading).

**Comment by fzaninotto** [comment]
Then you should add it to the list of public components (in `docs/Referecne.md`) and document it (probably in `docs/Theming.md`).

**Comment by tkvw** [comment]
Done

File: docs/Theming.md [hunk 1]
```diff
@@ -486,3 +486,22 @@ You can override the notification duration by setting the `autoHideDuration` pro
 ```
 
 **Tip**: if you use the `showNotification` action, then you can define `autoHideDuration` per message as the third parameter of the `showNotification` action creator.
+
+
+## Loading
+
+For consistency the `Loading` component of `react-admin` is exported.
```
Comment (djhi) [inline docs/Theming.md]: A description of the component's behavior (display a circular progress with optional messages, maybe add ing a link the MUI component too) and utility (display the same loading component as `react-admin` on custom pages for consistency)  would be better.

**Comment by djhi** [comment]
Thanks!

**Pull request merged**